src/foam/nanos/u2/navigation/BusinessLogoView.js

```diff
@@ -20,64 +20,71 @@ foam.CLASS({
   name: 'BusinessLogoView',
   extends: 'foam.u2.View',
 
-  imports: [ 'logo' ],
+  imports: [ 'group' ],
 
   documentation: 'View to display business logo and name.',
 
-  axioms: [
-    foam.u2.CSS.create({
-      code: function CSS() {/*
-        ^{
-          width: 175px;
-          display: inline-block;
-          text-align: center;
-          padding-top: 3px;
-          padding-left: 25px;
-        }
-        ^ img {
-          height: 30px;
-          padding-top: 10px;
-        }
-        ^ span{
-          position: relative;
-          font-weight: 300;
-          font-size: 16px;
-          margin-left: 10px;
-        }
-        ^business-name{
-          width: 70%;
-          text-align: left;
-          overflow: hidden;
-          text-overflow: ellipsis;
-          position: relative;
-          white-space: nowrap;
-          top: -35;
-          height: 20px;
-          display: inline-block;
-          vertical-align: middle;
-          margin-top: 32px;
-          margin-left: 5px;
-        }
-        ^placeholder-business{
-          width: 40px;
-          height: 40px;
-          margin: 5px;
-          border-radius: 50%;
-          background: white;
-        }
-      */}
-    })
-  ],
+  css: `
+    ^{
+      width: 175px;
+      display: inline-block;
+      text-align: center;
+      padding-top: 3px;
+      padding-left: 25px;
+    }
+    ^ img {
+      height: 30px;
+      padding-top: 10px;
+      cursor: pointer;
+    }
+    ^ span{
+      position: relative;
+      font-weight: 300;
+      font-size: 16px;
+      margin-left: 10px;
+    }
+    ^business-name{
+      width: 70%;
+      text-align: left;
+      overflow: hidden;
+      text-overflow: ellipsis;
+      position: relative;
+      white-space: nowrap;
+      top: -35;
+      height: 20px;
+      display: inline-block;
+      vertical-align: middle;
+      margin-top: 32px;
+      margin-left: 5px;
+    }
+    ^placeholder-business{
+      width: 40px;
+      height: 40px;
+      margin: 5px;
+      border-radius: 50%;
+      background: white;
+    }
+  `,
 
   methods: [
     function initE() {
-      var self = this;
+      var self     = this;
+      var logoSlot = this.group.logo$.map(function(logo) { return logo || self.logo; });
 
       this
         .addClass(this.myClass())
         .start()
-          .start({ class: 'foam.u2.tag.Image', data: this.logo }).end()
+          .tag({class: 'foam.u2.tag.Image', data$: logoSlot})
+          .on('click', self.goToDefault)
         .end();
     }
+  ],
+
+  listeners: [
```

Comment:
What is this for?

Reply:
I accidentally pushed my change for a different ticket to this PR just now. Will remove the BusinessLogoView.js changes.